content/blog/using-jlink-in-dockerfiles/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using Jlink in Dockerfiles instead of a JRE
+date: "2021-10-18T12:00:00+00:00"
+author: georgeadams
+description: How to swap a JRE docker image for a jlink runtime
+tags:
+  - Temurin
+  - Docker
+---
+
+The Eclipse Temurin project is excited to announce that the official docker images for Temurin binaries are now available on [Dockerhub](https://hub.docker.com/_/eclipse-temurin).
+
+If you were using a JDK image with Ubuntu Focal or Centos 7 as the base, the migration path is very simple. Simply change `adoptopenjdk` to `eclipse-temurin` and set the distro after the version number in the tag. E.g for Ubuntu Focal:
+
+```dockerfile
+FROM eclipse-temurin:11-focal
+# Continue with your application deployment
+RUN mkdir /opt/app
+COPY japp.jar /opt/app
+CMD ["java", "-jar", "/opt/app/japp.jar"]
+```
+
+If you were using a base image that was not Ubuntu Focal or Centos 7 you can use the copy command to bring the JDK into your image. E.g to use a Temurin binary inside a debian base image:
+
+```dockerfile
+FROM debian
+ENV JAVA_HOME=/opt/java/openjdk
+COPY --from=eclipse-temurin:11 $JAVA_HOME $JAVA_HOME
+ENV PATH="${JAVA_HOME}/bin:${PATH}"
+# Continue with your application deployment
+RUN mkdir /opt/app
+COPY japp.jar /opt/app
+CMD ["java", "-jar", "/opt/app/japp.jar"]
+```
+
+## What about JRE base images?
+
+The Eclipse Temurin project is currently planning on producing JRE images for OpenJDK8, these will be availble soon. For JDK11+ we recommend using `jlink` to produce a custom runtime:
```

Comment:
```suggestion
The Eclipse Temurin project is currently planning on producing JRE images for version 8, these will be [available soon](https://adoptium.net/releases.html?variant=openjdk8&jvmVariant=hotspot). For JDK 11+ it is possible to use `jlink` and produce a custom runtime that works directly with your application:
```
Just tweaked the intent here a bit, since the topic of JREs for Temurin 11 is still open; and many people have expressed an interest in having a "JRE" image layer that is fit for running multiple applications.